translate/storage/test_cpo.py

```diff
@@ -111,12 +109,9 @@ def test_merge_duplicates_msgctxt(self):
     def test_merge_blanks(self):
         """checks that merging adds msgid_comments to blanks"""
         posource = '#: source1\nmsgid ""\nmsgstr ""\n\n#: source2\nmsgid ""\nmsgstr ""\n'
-        pofile = self.poparse(posource)
-        assert len(pofile.units) == 2
+        pofile = self.poparse(posource, duplicatestyle="allow")
         pofile.removeduplicates("merge")
         assert len(pofile.units) == 2
-        print(pofile.units[0].msgidcomments)
-        print(pofile.units[1].msgidcomments)
```

Comment:
Please do not remove these print calls. Perhaps they are not necessary, but unless they are strictly necessary to be removed this is a cleanup that doesn't belong to this commit and makes more difficult to review the actual changes.